src/gui/lineedit/src/lineedit.h

```diff
@@ -12,26 +12,29 @@
 
 #include <QLineEdit>
 
-QT_BEGIN_NAMESPACE
 class QToolButton;
-QT_END_NAMESPACE
 
 class LineEdit : public QLineEdit
 {
   Q_OBJECT
 
 public:
-  LineEdit(QWidget *parent = 0);
```

Comment:
IMO, it's bad practice to default to 0, I'm open to other ideas.
By making it explicit, (sometimes) remind the developer to pass the correct pointer, e.g. not use `new LineEdit()` mindlessly.